src/NServiceBus.AcceptanceTests/Mutators/When_using_outgoing_tm_mutator.cs

```diff
@@ -39,7 +39,6 @@ public Task MutateOutgoing(MutateOutgoingTransportMessageContext context)
                 {
                     context.OutgoingHeaders["HeaderSetByMutator"] = "some value";
                     context.OutgoingHeaders[Headers.EnclosedMessageTypes] = typeof(MessageThatMutatorChangesTo).FullName;
-                    context.OutgoingBody = Encoding.UTF8.GetBytes("<MessageThatMutatorChangesTo/>");
```

Comment:
I had a talk with @timbussmann about this. Didn't look myself, but according to Tim it's really hard to achieve this.

About the invocation of the handler, that's still working because of https://github.com/Particular/NServiceBus/blob/RemoveXmlFromAcceptanceTests/src/NServiceBus.AcceptanceTests/Mutators/When_using_outgoing_tm_mutator.cs#L41